src/test/lib/article/tag-utils.test.ts

```
@@ -0,0 +1,191 @@
+    expect(filteredPosts[0].data.title).toBe("Getting Started Guide");
+  });
+
+  it("should filter posts by Comparison tag", () => {
+    const filteredPosts = filterPostsByTag(mockPosts as any, "Comparison", mockTagTranslations);
+    
+    expect(filteredPosts).toHaveLength(1);
+    expect(filteredPosts[0].data.title).toBe("Performance vs Speed");
+  });
+
+  it("should filter posts by Best Practices tag", () => {
+    const filteredPosts = filterPostsByTag(mockPosts as any, "Best Practices", mockTagTranslations);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)